app/page/home/activity.tsx

```diff
@@ -51,182 +53,174 @@ const UserOpHistoryItem: React.FC<{
-        <div className="flex-grow flex flex-col items-start">
-          {/* Status and token amount */}
-          <div className="flex items-center mb-[4px]">
-            <Clock className="w-[16px] h-[16px] mr-[8px]" />
-            <div className="mr-[8px] leading-[19px] text-[16px] text-[#000000]">
-              {status}
-            </div>
-            <div className="mr-[8px] leading-[19px] text-[16px] text-[#000000]">
-              1.2
-            </div>
-            <div className="leading-[19px] text-[16px] text-[#000000]">ETH</div>
-          </div>
-          {/* To address */}
-          <div className="leading-[19px] text-[16px] text-[#bbbbbb] whitespace-nowrap">
-            to: 0x0000...00000
+const TokenLog: React.FC<{
+  tokenName: string
+  value: BigNumberish
+  toAddress: HexString
+  txStatus: TransactionStatus
+}> = ({ tokenName, value, toAddress, txStatus }) => {
+  return (
+    <div className="w-full flex items-center py-[14px]">
+      {/* Status and to address */}
+      <div className="flex-grow flex flex-col items-start">
+        {/* Status and token amount */}
+        <div className="flex items-center mb-[4px] gap-[8px]">
+          <ArrowUpRight className="w-[16px] h-[16px]" />
+          <div className="text-[16px] text-[#000000]">Send</div>
+          <div className="text-[16px] text-[#000000]">
+            {number.formatUnitsToFixed(value, 18, 2)}
           </div>
+          <div className="text-[16px] text-[#000000]">{tokenName}</div>
+          {/* Status mark */}
+          <StatusMark txStatus={txStatus} />
         </div>
-        {/* Transaction time */}
-        <div>
-          <div className="flex flex-col items-end">
-            <div className="mb-[4px] leading-[19px] text-[16px] text-[#000000]">
-              {creationDate}
-            </div>
-            <div className="leading-[19px] text-[16px] text-[#000000]">
-              {creationTime}
-            </div>
-          </div>
+        {/* To address */}
+        <div className="text-[16px] text-[#bbbbbb] whitespace-nowrap">
+          to: {address.ellipsize(toAddress)}
         </div>
       </div>
-    )
-  }
+    </div>
+  )
+}
 
-  // Others' case
+const ContractLog: React.FC<{
+  toAddress: HexString
+  txStatus: TransactionStatus
+}> = ({ toAddress, txStatus }) => {
   return (
-    <div className="w-full flex items-center p-[14px_0px_14px_0px]">
+    <div className="w-full flex items-center py-[14px]">
       {/* Status and to address */}
       <div className="flex-grow flex flex-col items-start">
         {/* Status and token amount */}
         <div className="flex items-center mb-[4px]">
-          <CircleQuestion className="w-[16px] h-[16px] mr-[8px]" />
-          <div className="mr-[8px] leading-[19px] text-[16px] text-[#000000]">
-            {status}
+          <ArrowRightArrowLeft className="w-[16px] h-[16px] mr-[8px]" />
+          <div className="mr-[8px] text-[16px] text-[#000000]">
+            Interact with Dapp
```

Comment:
```suggestion
            Interact with contract
```

Reply:
It might have been changed to `Contract interaction`.